db/migrate/20170313063156_create_developers.rb

```diff
@@ -0,0 +1,15 @@
+class CreateDevelopers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :developers do |t|
+      t.string :name
+      t.timestamps
+    end
+
+    change_table :games do |t|
+      t.references :developer, foreign_key: true
+    end
+
+    add_index :games, :title
+    add_index :games, [:developer_id, :name], :unique => true
```

Comment:
Please change :name to :title.

Don't forget to migrate to the DB before making a PR. These small mistakes can easily be avoided that way!